.gitignore

```diff
@@ -18,4 +18,8 @@ go.work*
 pkg/aws/services/gomock_reflect_*
 
 # Image build tarballed bundles
-*.tgz
\ No newline at end of file
+*.tgz
+
+# generated during make-deploy
+tls.crt
+tls.key
```

Comment:
I think it is better to remove them at the end of Makefile

Reply:
Good idea. WIll change this.

Reply:
added ```rm``` to makefile, removed from ```.gitignore```